usr/src/uts/armv8/os/mlsetup.c

```diff
@@ -197,6 +199,12 @@ mlsetup(struct regs *rp)
 	boot_ncpus = NCPU;
 	max_ncpus = boot_max_ncpus = boot_ncpus;
 
+	/*
+	 * Initialise CPU info for the boot processor and fill in accurate
+	 * values for boot_ncpus, boot_max_ncpus and max_ncpus.
+	 */
+	cpuinfo_bootstrap(CPU);
```

Comment:
Is it safe to do this after we switched to our own exception vector?  That is, is code used by our exception vector (while panicking, most especially) going to work before this is initialized?

(you can stick `*((int64_t *)NULL) = 1` just before this line, temporarily, to make it panic before then, and see what happens)

Reply:
https://gist.github.com/r1mikey/4d6f3bdd6b2b41b8eb2908bbe29606fe

Not ideal, as PSCI isn't yet initialised - I'll separate the PSCI configs soon (similar to this PR), make them safe to bootstrap early on and move PSCI initialisation much earlier when PSCI is present. That'll my next PR.